merge/mergeRaw.go

```diff
@@ -120,7 +122,21 @@ func Combine(src *Merge, dst *Merge) *Merge {
 					d.Value = s.Value
 				}
 			}
+			override(d, src.conf)
 		}
 	}
 	return dst
 }
+
+// override the value if its key is in the conf.ConfigOverrides
+func override(d *Value, conf c.Conf) {
+	for k, v := range conf.ConfigOverrides {
+		if strings.Contains(d.Key, k) {
+			var buffer bytes.Buffer
+			buffer.WriteString(string([]rune(d.Value)[0]))
+			buffer.WriteString(v)
+			d.Value = buffer.String()
+			return
+		}
+	}
+}
```

Comment:
Would it be really difficult to tie the file name that you're merging into the MergeForward config?

Not seeing a config file; I'm assuming that this is a straight "key match", which might bite us if we mean to over-write a specific field in one config file that happens to have the same name in another.

We have so many fields, and so many developers, that this is a bug that is probably going to creep up.

Having the file name be part of it narrows the scope